diff/diff_test.go

```diff
@@ -103,27 +102,33 @@ func TestParseFileDiffHeaders(t *testing.T) {
 			filename: "sample_file.diff",
 			wantDiff: &FileDiff{
 				OrigName: "oldname",
-				OrigTime: &pbtypes.Timestamp{Seconds: 1255273940},
+				// Timestamp{Seconds: 1255273940}
+				OrigTime: []byte{1, 0, 0, 0, 14, 194, 99, 236, 212, 0, 0, 0, 0, 255, 255},
```

Comment:
```suggestion
				OrigTime: []byte{1, 0, 0, 0, 14, 194, 99, 236, 212, 0, 0, 0, 0, 255, 255}, // 2009-10-11 15:12:30
```

Same for the other timestamp comments in this file.